WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyContentView.swift

```diff
@@ -268,19 +271,40 @@ final class SiteAssemblyContentView: UIView {
         ])
     }
 
+    private func installErrorStateView() {
+        guard let errorStateView = errorStateView else {
+            return
+        }
+
+        errorStateView.alpha = 0
+        addSubview(errorStateView)
+
+        NSLayoutConstraint.activate([
+            errorStateView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            errorStateView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            errorStateView.topAnchor.constraint(equalTo: topAnchor),
+            errorStateView.bottomAnchor.constraint(equalTo: bottomAnchor),
+        ])
+    }
+
     private func layoutIdle() {
         completionLabel.alpha = 0
         statusStackView.alpha = 0
+        errorStateView?.alpha = 0
```

Comment:
I know it is just a one liner, but do you think it would make sense to encapsulate this in two methods, reading `showErrorView` and `hideErrorView`

Reply:
Thanks for the suggestion; I tend to employ a two-line threshold for method refactoring.